dao/src/main/java/org/thingsboard/server/dao/sql/query/DefaultAlarmQueryRepository.java

```diff
@@ -68,6 +68,7 @@ public class DefaultAlarmQueryRepository implements AlarmQueryRepository {
         alarmFieldColumnMap.put("originatorId", ModelConstants.ALARM_ORIGINATOR_ID_PROPERTY);
         alarmFieldColumnMap.put("originatorType", ModelConstants.ALARM_ORIGINATOR_TYPE_PROPERTY);
         alarmFieldColumnMap.put("originator", "originator_name");
+        alarmFieldColumnMap.put("originatorLabel", ModelConstants.ALARM_ORIGINATOR_LABEL_PROPERTY);
     }
 
     private static final String SELECT_ORIGINATOR_NAME = " COALESCE(CASE" +
```

Comment:
Ideally, this statement should have the cases for any entity that has name field